### DISCUSSION TITLE
Does it mean i'm accepted

### DISCUSSION THREAD

**Discussion by samietadam** [community/community#123934] [GitHub Education]
### Hi everyone,

Hello, I had a link to check the status of my request but I don't know if that means I am accepted? 
![Capture d’écran du 2024-05-15 13-02-17](https://github.com/community/community/assets/97306814/25539869-1cac-49e8-b87f-56f11fd594ac)

**Comment by Aditya020224** [comment]
once you have submitted it and got link then you rest assure

**Reply by samietadam** [reply]
that's mean that i'm not accepted?

**Reply by Aditya020224** [reply]
no, you have to just wait for some hours or days to get finally approved message so do not worry just rest assure okay

**Comment by davevad93** [comment]
Hi @samietadam ,

That means your request has been approved, but you still have to wait a few days for your benefits to become available. When the student benefits are finally available the `approved` green badge will turn purple and you'll see the following message:

`Congrats! Your request for @samietadam was approved and your benefits are now available.`
